test/dense/svds/gesvd/magmaDgesvd.jl

```diff
@@ -0,0 +1,40 @@
+using MAGMA
+using MAGMA.Utilities
+using MAGMA.Dense
```

Comment:
remove `Dense` module as well. If we find it is necessary, we can always add it later, but we only want a functional svd for now. Don't do any extra work.

Reply:
Both removed.